scala-collector/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM java:8-alpine
+
+ENV SCALA_COLLECTOR=20170611 \
+BUILD_PACKAGES='bash curl-dev git curl ca-certificates gettext' \
+SBT_VERSION=0.13.15 \
```

Comment:
this fails 'wget https://github.com/sbt/sbt/releases/download/v13.0/sbt-13.0.tgz  ' can you give me a valid download link for `0.13.0`?